src/cpp/thread_manager/thread_manager.h

```diff
@@ -72,7 +72,7 @@ class ThreadManager {
   // The implementation of DoWork() should also do any setup needed to ensure
   // that the next call to PollForWork() (not necessarily by the current thread)
   // actually finds some work
-  virtual void DoWork(void* tag, bool ok) = 0;
+  virtual void DoWork(void* tag, bool ok, bool resources) = 0;
```

Comment:
Can you update the comment above to talk about `resources` parameter too ?

Reply:
Thanks! Done.